clmm/galaxycluster.py

```diff
@@ -297,7 +289,7 @@ def compute_tangential_and_cross_components(
             dec_lens=self.dec,
             geometry=geometry,
             validate_input=self.validate_input,
-            coordinate_system=self.coordinate_system,
+            coordinate_system=self.galcat.meta["coordinate_system"],
```

Comment:
Need to put this in the documentation of the function.

Reply:
Resolved